Script/AtomicEditor/ui/modal/build/platforms/AndroidSettingsWidget.ts

```diff
@@ -181,7 +181,7 @@ class AndroidSettingsWidget extends Atomic.UIWidget implements BuildSettingsWind
         if (this.antPathEdit.text.length && this.antPathEdit.text != ToolCore.toolEnvironment.toolPrefs.antPath)
             ToolCore.toolEnvironment.toolPrefs.antPath = this.antPathEdit.text;
 
-        this.settings.sDKVersion = this.sdkTargetSelect.text;
+        this.settings.sdkVersion = this.sdkTargetSelect.text;
```

Comment:
Man, I remember having to type this and thinking wish I had time to fix it... that time is now, thanks!